docs/rfc-1-staking-contract.adoc

```diff
@@ -277,27 +277,27 @@ out of the scope of this document.
 
 === Delegating a stake
 
-==== `stake(address operator, address beneficiary, address authorizer, uint96 amount) external` 
+==== `stake(address stakingProvider, address beneficiary, address authorizer, uint96 amount) external` 
    
-Creates a delegation with `msg.sender` owner with the given operator,
+Creates a delegation with `msg.sender` owner with the given provider,
 beneficiary, and authorizer. Transfers the given amount of T to the staking
 contract. The owner of the delegation needs to have the amount approved to
 transfer to the staking contract.
 
-==== `stakeKeep(address operator) external`
+==== `stakeKeep(address stakingProvider) external`
 
 Copies delegation from the legacy KEEP staking contract to T staking contract.
 No tokens are transferred. Caches the active stake amount from KEEP staking 
 contract. Can be called by anyone.
 
-==== `stakeNu(address operator, address payable beneficiary, address authorizer) external`
+==== `stakeNu(address stakingProvider, address payable beneficiary, address authorizer) external`
 
 Copies delegation from the legacy NU staking contract to T staking contract,
 additionally appointing beneficiary and authorizer roles. Caches the amount
 staked in NU staking contract. Can be called only by the original delegation
 owner.
 
-==== `refreshKeepStakeOwner(address operator) external onlyOwnerOf(operator)`
+==== `refreshKeepStakeOwner(address stakingProvider) external onlyOwnerOf(stakingprovider)`
```

Comment:
mismatched camel case for `stakingProvider` vs `stakingprovider`? If so, other places lower down as well.

Reply:
✔️